andela/ah-frontend-space
# #167399543 users should be able to delete articles

#### What does this PR do?

- Enable a user to delete an article

#### Description of Task to be completed?

- Add a delete button to the article detail page
- Make a call to the delete endpoint and ensure an article is deleted when a user clicks delete article
- Return relevant responses to the user during the delete process

#### How should this be manually tested?

- visit ` https://ah-frontend-space-stagin-pr-25.herokuapp.com/  `to view all the articles
- ensure you are logged in
- click readmore to view the article details of the article that you created
- there will be a delete button to enable you delete an article

#### What are the relevant pivotal tracker stories?

[#167399543](https://www.pivotaltracker.com/story/show/167399543)

#### Screenshots (if appropriate)

<img width="956" alt="delete" src="https://user-images.githubusercontent.com/40593659/61931257-c3fc0f80-af88-11e9-9070-7b43a325200a.png">
